File: src/main/java/com/examples/testing/withdeadcodeannotation/Question.java
package com.examples.testing.withdeadcodeannotation;

import edu.illinois.cs.cs125.questioner.lib.Correct;

/*
 * Testing dead code comments.
 */
@Correct(name = "With Dead Code Annotation", version = "2023.2.0", author = "challen@illinois.edu", focused = true)
public class Question {
  public static int whatever(int value) {
    if (value % 2 == 0) {
      return value * 2;
    }
    assert false;
    // mutate-disable
    return -1; // dead code
  }
}

File: src/main/java/com/examples/testing/withdeadcodeannotation/correct/kotlin/Question.kt
@file:AlsoCorrect

package com.examples.testing.withdeadcodeannotation.correct.kotlin

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect

/*
 * Testing dead code comments.
 */
fun whatever(value: Int): Int {
  if (value % 2 == 0) {
    return value * 2
  }
  assert(false)
  // mutate-disable
  return -1 // dead code
}